docs/source/configuration/telemetry/instrumentation/spans.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Spans
 subtitle: Add router lifecycle context to traces
-description: Use spans to add contextual information from the Apollo Router to traces displayed by your application performance monitors (APM).
+description: Use spans to add contextual information from the Apollo GraphOS Router to traces displayed by your application performance monitors (APM).
```

Comment:
Available in both Router Core and GraphOS Router (for all occurrences on this page). @BrynCooke - there are no custom spans right?